python_modules/dagster/dagster/components/core/defs_module.py

```diff
@@ -88,28 +96,45 @@ def _add_defs_yaml_code_reference_to_spec(
 
 
 class CompositeYamlComponent(Component):
-    def __init__(self, components: Sequence[Component], source_positions: Sequence[SourcePosition]):
+    def __init__(
+        self,
+        components: Sequence[Component],
+        source_positions: Sequence[SourcePosition],
+        post_processors_list: Sequence[Sequence[AssetPostProcessor]],
```

Comment:
Right now this means we can't put post-processors on a Python component, right? How would you envision attaching them in that case, e.g. would it be in the decorator?